sdk/eventhub/Azure.Messaging.EventHubs/src/Amqp/AmqpConnectionScope.cs

```diff
@@ -76,6 +76,13 @@ internal class AmqpConnectionScope : IDisposable
         ///
         private static TimeSpan MinimumAuthorizationRefresh { get; } = TimeSpan.FromMinutes(2);
 
+        /// <summary>
+        ///   The maximum amount of time to allow before authorization is refreshed; any calculations
+        ///   that call for refreshing less frequently will be substituted with this value.
+        /// </summary>
+        ///
+        private static TimeSpan MaximumAuthorizationRefresh { get; } = TimeSpan.FromHours(2);
```

Comment:
How did we choose 2 hours, if the limit was 49 days?

Reply:
Arbitrary selection.  It seemed to be a reasonable approach to refresh authorization every 2 hours whether there was danger of it expiring or no.   There's no reason to not to so - the service is happy to accept the refresh and the interval is long enough that it isn't putting any strain on the client or service with frequent network chatter.

Reply:
I think it may be better to be more in sync with whatever the token authorization duration is. We have to make it less than 49 days due to a technical limitation, but I'm wary about completely ignoring anything above 2 hours.

Reply:
Why?

Reply:
Coordinated with Josh offline;  we agreed the value is arbitrary either way but decided to adopt a value closer to the limit to further understanding.